### ISSUE TITLE
hi lee!

### ISSUE THREAD

**Issue by lq677** [JintaoLee-Roger/SeismicSuperResolution#1]
Sorry to bother you. Recently, I have read your paper and very interested in it. But I have some questions.Thank you!

Your research is based on seismic data, right? Not on images in the traditional sense, such as JEPG, BMP and so on.

I'm looking forward to your reply. Thanks!

**Comment by JintaoLee-Roger** [comment]
> 

Yes, it is based on seismic data (except Figure 11). You can notice that the range of amplitude is different from the images in JEPG format (BMP and so on) .

**Comment by lq677** [comment]
Your answer really helped me a lot. Thank you very much for your reply!
